cpp/tests/traversal/legacy/sssp_test.cu

```diff
@@ -108,11 +108,8 @@ void ref_sssp(const std::vector<MaxEType>& rowPtr,
   }
 }
 
-// do the perf measurements
-// enabled by command line parameter s'--perf'
-static int PERF = 0;
-
 // iterations for perf tests
+
 // enabled by command line parameter '--perf-iters"
```

Comment:
This comment is no longer true, correct?   Probably should at least delete the comment if we want to leave in the `run 5 times` logic.

Reply:
Correct, and done.